libavcodec/vvc/vvc_ctu.c

```diff
@@ -865,8 +865,12 @@ static enum IntraPredMode derive_center_luma_intra_pred_mode(const VVCFrameConte
     const int cu_pred_mode        = SAMPLE_CTB(fc->tab.cpm[0], x_center, y_center);
     const int intra_pred_mode_y   = SAMPLE_CTB(fc->tab.ipm, x_center, y_center);
 
-    if (intra_mip_flag)
-        return INTRA_PLANAR;
+    if (intra_mip_flag) {
+        if (cu->tree_type == SINGLE_TREE && sps->chroma_format_idc == CHROMA_FORMAT_444)
+            return -1;
```

Comment:
how about define INTRA_INVALID

Reply:
Done in 71c1841716c4b7c8778f54a6b9080856577cbac4.

I think `INTRA_DEFAULT` is a little more fitting as `-1` is now valid - see change to table 20 in spec fix.